From aaa2645bcc70f364657da4de39c9633029ef3b4b Mon Sep 17 00:00:00 2001
From: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
Date: Tue, 17 Oct 2023 14:41:56 +0200
Subject: [PATCH 0771/1002] configs: rpi: Compile OV64A40 and BU64754 as
 modules

Compile both the OV64A40 and the BU64754 drivers used by the Arducam
64MP camera module based as modules in the defconfig files for

- rpi5
- rpi4
- rpi2/3
- rpi1/0

Signed-off-by: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
---
 arch/arm/configs/bcm2709_defconfig   | 2 ++
 arch/arm/configs/bcm2711_defconfig   | 2 ++
 arch/arm/configs/bcmrpi_defconfig    | 2 ++
 arch/arm64/configs/bcm2711_defconfig | 2 ++
 arch/arm64/configs/bcm2712_defconfig | 2 ++
 arch/arm64/configs/bcmrpi3_defconfig | 2 ++
 6 files changed, 12 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 360ea2acbaa8..1ccd77797301 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -923,11 +923,13 @@ CONFIG_VIDEO_IMX708=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_VIDEO_OV2311=m
 CONFIG_VIDEO_OV5647=m
+CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
+CONFIG_VIDEO_BU64754=m
 CONFIG_VIDEO_DW9807_VCM=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
 CONFIG_VIDEO_UDA1342=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index d2e6065d4dc5..13015b721794 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -938,11 +938,13 @@ CONFIG_VIDEO_IMX708=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_VIDEO_OV2311=m
 CONFIG_VIDEO_OV5647=m
+CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
+CONFIG_VIDEO_BU64754=m
 CONFIG_VIDEO_DW9807_VCM=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
 CONFIG_VIDEO_UDA1342=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 22a2c6d856c0..098a611daf17 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -917,11 +917,13 @@ CONFIG_VIDEO_IMX708=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_VIDEO_OV2311=m
 CONFIG_VIDEO_OV5647=m
+CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
+CONFIG_VIDEO_BU64754=m
 CONFIG_VIDEO_DW9807_VCM=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
 CONFIG_VIDEO_UDA1342=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index f08d3eac4ad7..90a33e9c337b 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -972,11 +972,13 @@ CONFIG_VIDEO_IMX708=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_VIDEO_OV2311=m
 CONFIG_VIDEO_OV5647=m
+CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
+CONFIG_VIDEO_BU64754=m
 CONFIG_VIDEO_DW9807_VCM=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
 CONFIG_VIDEO_UDA1342=m
diff --git a/arch/arm64/configs/bcm2712_defconfig b/arch/arm64/configs/bcm2712_defconfig
index 22b245277806..a1cd4f8299f7 100644
--- a/arch/arm64/configs/bcm2712_defconfig
+++ b/arch/arm64/configs/bcm2712_defconfig
@@ -975,11 +975,13 @@ CONFIG_VIDEO_IMX708=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_VIDEO_OV2311=m
 CONFIG_VIDEO_OV5647=m
+CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
+CONFIG_VIDEO_BU64754=m
 CONFIG_VIDEO_DW9807_VCM=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
 CONFIG_VIDEO_UDA1342=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 617b0040b47a..e05bcc641be6 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -894,11 +894,13 @@ CONFIG_VIDEO_IMX708=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_VIDEO_OV2311=m
 CONFIG_VIDEO_OV5647=m
+CONFIG_VIDEO_OV64A40=m
 CONFIG_VIDEO_OV7251=m
 CONFIG_VIDEO_OV7640=m
 CONFIG_VIDEO_OV9282=m
 CONFIG_VIDEO_AD5398=m
 CONFIG_VIDEO_AK7375=m
+CONFIG_VIDEO_BU64754=m
 CONFIG_VIDEO_DW9807_VCM=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
 CONFIG_VIDEO_UDA1342=m
-- 
2.44.0

